frontend/src/concepts/pipelines/content/import/PipelineImportModal.tsx

```diff
@@ -121,8 +126,11 @@ const PipelineImportModal: React.FC<PipelineImportModalProps> = ({ isOpen, onClo
                 data-testid="pipeline-name"
                 name="pipeline-name"
                 value={name}
-                onChange={(e, value) => setData('name', value)}
+                onChange={(_e, value) => setData('name', value)}
+                validated={hasDuplicateNameError ? 'error' : 'default'}
               />
+
+              {hasDuplicateNameError && <DuplicateNameHelperText name={name} isError />}
```

Comment:
why are we showing an error state here? i thought it was just info. we are not blocking the user from doing this just informing them

Reply:
The user is unable to create duplicate pipeline names within the same project. That results in an API error, so no, for this modal, and the version modal, it is considered an error, and we're catching it before they experience that API error.

Reply:
do we block the submit?

Reply:
With this PR, if there is a duplicate name identified, we are preventing the ability to submit the modal. If the user were to click submit without a duplicate name identified, an API error would show in the modal and the user would be forced to enter a new name regardless.

Reply:
to keep with consistency across our forms, we should just use the info version and not block them from submitting and let the api throw an error. I believe will eventually start to show errors on fields but not until we adopt a form lib to do it.

Reply:
thats true, but i believe consistency efforts are covered by someone else in UXD. So im not sure if everything is always in sync